examples/dao/assets/proposal-lsig.py

```diff
@@ -0,0 +1,56 @@
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def proposal_lsig(ARG_OWNER):
+    """
+    Represents Proposal Lsig (a contract account owned by the proposer)
+    """
+
+    # check no rekeying, close remainder to, asset close to for a txn
```

Comment:
below we are checking for `rekey_to` only.

Reply:
updated